planning/motion_velocity_planner/autoware_motion_velocity_planner_common/test/test_polygon_utils.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright 2025 Tier IV, Inc.
```

Comment:
In case of, ... something
```suggestion
// Copyright 2025 TIER IV, Inc.
```